iron-dropdown.html

```diff
@@ -190,6 +210,8 @@
             target = this._positionRect.left;
           }
 
+          target += this.horizontalOffset;
```

Comment:
Hmm so it should be negative for `left`? Maybe that should go in the comment too :(